src/content/plugins/module-federation-plugin.mdx

```diff
@@ -2,14 +2,15 @@
 title: ModuleFederationPlugin
 contributors:
   - XiaofengXie16
-
 related:
   - title: Module Federation
     url: https://webpack.js.org/concepts/module-federation/
 ---
 
 The `ModuleFederationPlugin` allows a build to provide or consume modules with other independent builds at runtime.
 
+<StackBlitzPreview example="module-federation?terminal=" />
```

Comment:
I believe we need to extend this [`StackBlitzPreview`](https://github.com/webpack/webpack.js.org/blob/master/src/components/StackBlitzPreview/StackBlitzPreview.jsx) component:

![image](https://user-images.githubusercontent.com/1091472/121831046-19667480-ccf9-11eb-9e2d-1b84bd9aa3a2.png)

`Check out this guide live on StackBlitz.` doesn't make sense on this page because there's no guide.

Something like below would be great in my opinion:

```jsx
<StackBlitzPreview example="module-federation?terminal=" description="Check out a federation module example" />
```

Reply:
Updated. I am sorry for the inconvenience.  I will keep in mind regarding the PR rules in my future PRs. Again, thank you for the feedback. Please let me know if there are any additional changes needed. 😃

Reply:
Don't worry, we all make mistakes.